src/assets/thirdPartyServices/index.js

```diff
@@ -1,7 +1,7 @@
 import React, { lazy } from "react";
 
 const ExampleService = lazy(() => import("./snet/example_service"));
-const CNTKImageRecognition = lazy(() => import("./CNTKImageRecognition.js"));
+const CNTKImageRecognition = lazy(() => import("./snet/cntk-image-recon"));
```

Comment:
We could stick to underscore naming conventions. This would translate to `cntk_image_recon`.

Reply:
[done](https://github.com/singnet/refactored-dapp/pull/120/commits/27f23b05e26ce13b7938be2bae1a5c129ef979b5)